src/Tests/Microsoft.Diagnostics.Monitoring.Tool.FunctionalTests/EgressTests.cs

```diff
@@ -285,6 +287,15 @@ public async Task EgressNotExistTest()
                 });
         }
 
+        public static bool IsNotCore31OnOSX()
+        {
+#if NET5_0_OR_GREATER
+            return true;
+#else
+            return RuntimeInformation.IsOSPlatform(OSPlatform.OSX);
```

Comment:
Should this be inverted? Unless I'm reading it wrong, this returns true when we're on NetCore31 and the OSPlatform is `OSX`, but the function name is `IsNotCore31OnOSX`